examples/03-http-any.php

```diff
@@ -22,16 +22,13 @@
         function (Psr\Http\Message\ResponseInterface $response) use ($url) {
             // return only the URL for the first successful response
             return $url;
-        }
-    );
+        });
```

Comment:
Don't think this change is necessary

Reply:
ok changed it back t the origina for, thks @SimonFrings 👍